core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataManagerTest.scala

```diff
@@ -931,6 +932,42 @@ class GroupMetadataManagerTest {
     assertTrue(group.has(memberId))
   }
 
+  @Test
+  def testShouldThrowExceptionForUnsupportedGroupMetadataVersion(): Unit = {
```

Comment:
Is this test related to the change or a gap you identified? It's fine if it's the latter, just trying to understand.

Reply:
This test doesn't relate to the change. I just like to add missing tests for what I saw. :)